src/mongo_types.ts

```diff
@@ -1,4 +1,4 @@
-import type { Document, ObjectId } from './bson';
+import type { Binary, Document, ObjectId, BSONRegExp } from './bson';
```

Comment:
Let's talk through the changes in this file just for sanity's sake

Reply:
Talked through this